sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/FormRecognizerAsyncClient.java

```diff
@@ -242,10 +237,9 @@
      */
     @ServiceMethod(returns = ReturnType.COLLECTION)
     public PollerFlux<OperationResult, List<FormPage>>
-        beginRecognizeContentFromUrl(String formUrl, Duration pollInterval) {
-        final Duration interval = pollInterval != null ? pollInterval : DEFAULT_DURATION;
-        return new PollerFlux<OperationResult, List<FormPage>>(interval,
-            contentAnalyzeActivationOperation(formUrl),
+        beginRecognizeContentFromUrl(RecognizeOptions recognizeOptions) {
```

Comment:
Do we need this method? It's the same as [beginRecognizeContent(RecognizeOptions recognizeOptions)](https://github.com/Azure/azure-sdk-for-java/pull/12111/files#diff-f3e13c2bd95cecb8a90329a3c1e1f6dbR297) below.

Reply:
I think we should have ` beginRecognizeContentFromUrl(RecognizeOptions recognizeOptions)`
 too, since it allows the user an option to pass the configurable options when passing in a URL input for the form/receipt.

Reply:
updated.